TodoApp/src/TodoApp.Contract/Repositories/IItemRepository.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using TodoApp.Contract.Models;
 
 namespace TodoApp.Contract.Repositories
 {
     public interface IItemRepository
     {
-        Task<Item[]> GetAllAsync();
+        Task<List<Item>> GetAllAsync();
```

Comment:
Let's get back to `IEnumerable<Item>` here (see [this comment](https://github.com/LindaLangerova/kentico-onboarding-cs/pull/3#discussion-diff-191806205R10)) 😉